### PR TITLE
Hide pending deposits to Emburse in the global Emburse tx list

### PR DESCRIPTION
**Pull request by thesephist**
This is a PR to fix something Michael pointed out, where Emburse has started syncing Pending transactions for deposits from Bank accounts to Emburse accounts.

This change does two things:

1. In the global emburse transaction list, hide any transactions that are **pending** and **amount > 0** (are deposits), since they shouldn't be processed until they're complete, to be conservative.
2. Don't count any pending deposits as defined in (1) as "under review"

### CODE REVIEW

File: app/models/emburse_transaction.rb [hunk 1]
```diff
@@ -16,7 +16,17 @@ class EmburseTransaction < ApplicationRecord
   validates_uniqueness_of_without_deleted :emburse_id
 
   def under_review?
-    self.event_id.nil? && undeclined?
+    self.event_id.nil? && undeclined? && !should_be_ignored?
```
Comment (tmb) [inline app/models/emburse_transaction.rb]: You should just be able to do `event_id.nil?` here w/o the `self`

File: app/models/emburse_transaction.rb [hunk 2]
```diff
@@ -16,7 +16,17 @@ class EmburseTransaction < ApplicationRecord
...
+  end
+
+  # Emburse surfaces pending transactions of two types:
+  # 1. expenses, which are negative TXs signifying money spent on Emburse cards
+  # 2. deposits, which are TXs we make to move money from Bank to Emburse
+  #
+  # No. 2 (deposits) shouldn't really be triaged until they become COMPLETED. So
+  # we ignore them for the purposes of reviewing transactions.
+  def should_be_ignored?
+    :pending && self.amount > 0
```
Comment (tmb) [inline app/models/emburse_transaction.rb]: Same here, you should just be able to do `amount > 0`

**Review by tmb — CHANGES_REQUESTED** [review]
Otherwise looks good!

**Comment by thesephist** [comment]
Just gonna merge since it's minor and I think I addressed the changes 🤷‍♂️ sorry @tmb I robbed you of your opportunity to make things light up in green

**Comment by thesephist** [comment]
Undoing since it seems like there are unclear things about which things are syncing in production... we need a better way to be able to see the hidden transactions.

**Comment by thesephist** [comment]
(Note to self): We should also have the dashboard hide notifications for the hidden pending transactions in the root page.

**Pull request merged**